mesheryctl/internal/cli/root/system/error.go

```diff
@@ -58,3 +63,23 @@ func ErrUnmarshal(err error, obj string) error {
 func ErrUnsupportedPlatform(platform string, config string) error {
 	return errors.New(ErrUnsupportedPlatformCode, errors.Alert, []string{"the platform ", platform, " is not supported. Supported platforms are:\n\n- docker\n- kubernetes\n\nVerify this setting in your meshconfig at ", config, " or verify by executing `mesheryctl system context view`"}, []string{err.Error()}, []string{}, []string{})
 }
+
+func ErrRetrievingCurrentContext(err error) error {
+	return errors.New(ErrRetrievingCurrentContextCode, errors.Alert, []string{"Error retrieving current context"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrSettingTemporaryContext(err error) error {
+	return errors.New(ErrSettingTemporaryContextCode, errors.Alert, []string{"Error setting temporary context"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrCreateManifestsFolder(err error) error {
+	return errors.New(ErrCreateManifestsFolderCode, errors.Alert, []string{"Error creating manifest folder"}, []string{err.Error()}, []string{}, []string{})
```

Comment:
@nehaLohia27 thanks for this PR. 👏 

Will you please include a long description, probable cause, and suggested remediation for each of these errors? Examples can be seen on https://docs.meshery.io/reference/error-codes

Reply:
@leecalcote Do you want to include this change in doc link https://docs.meshery.io/reference/error-codes

Reply:
Hey @nehaLohia27 So over here in the errors.New(..) we are passing 6 arguments `errors.New(ErrExampleCode, errors.Alert, []string{"<short-description>"}, []string{"<long-description>"}, []string{"<probable-cause>"}, []string{"<suggested remediation>"})`. But in most of the case you are missing with the probable-cause and the suggested remediation so that our user know what is the error and how to solve it. As the mesheryctl  error codes will be the most trafficked error so we need to describe the error in details. And you do not need to update this in the docs manually because after this PR gets merged it will be automatically updated in the Meshery Docs.

Reply:
If you face anymore problem either you can hit me in the slack or you can find a good documentation here https://docs.meshery.io/project/contributing-error .

Reply:
ok @sayantan1413 got it. Thanks for the explaination.

Reply:
@leecalcote @metonymic-smokey I have added the error code references in error.go. Could you please verify once. Also I might have missed few functions , was not sure about the messages to add. 

Thanks
Neha Lohia